### DISCUSSION TITLE
.NET 8 为 WPF 带来了原生的 OpenFileDialog 及 OpenFolderDialog

### DISCUSSION THREAD

**Discussion by BYJRK** [BYJRK/DotNet-Discussions#163] [WPF 开发]
https://devblogs.microsoft.com/dotnet/wpf-file-dialog-improvements-in-dotnet-8/

https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-8#windows-presentation-foundation

也是有点离谱，居然现在才来

**Comment by xingzuhui** [comment]
传道\授业\解惑,十月寒流,可以为师矣.

**Reply by xingzuhui** [reply]
你讲解都是
1.先提出问题,再讲解决,很好,和zxjy不一样. 好多了. 但是往往是10个小时的内容,居然可以精简到1小时内.
2.英语发音准确.这个比很多老师好,好太多了,你知道只要我们妓院的院长才达到这水平.
3.问题深刻,不是浅尝辄止,浮光掠影.真真的极好的.继续做下去,让中国软件也发展一点儿,十月寒流配享太庙.
